Assets/Scripts/CutsceneCamera.cs

```diff
@@ -6,41 +7,72 @@ public class CutsceneCamera : MonoBehaviour
 {
     [CanBeNull] private static GameObject _targetObject;
     private static Vector2 _target;
+    private static Vector2 _lead;
+    private static Camera _camera;
+    private static float _cameraResizeSpeed;
+    private static float _cameraSize;
 
     [SerializeField] private float xSmoothTime;
     [SerializeField] private float ySmoothTime;
     private float _xVelocity;
     private float _yVelocity;
 
+    private void Awake()
+    {
+        _camera = GetComponent<Camera>();
+    }
+
     private void LateUpdate()
     {
-        if (_targetObject) _target = _targetObject.transform.position;
+        if (_targetObject)
+            _target = new Vector2(_targetObject.transform.position.x, _targetObject.transform.position.y) + _lead;
 
         // apply smoothing to the camera
-        var camPosition = transform.position;
-        var smoothedX = Mathf.SmoothDamp(camPosition.x, _target.x, ref _xVelocity, xSmoothTime);
-        var smoothedY = Mathf.SmoothDamp(camPosition.y, _target.y, ref _yVelocity, ySmoothTime);
+        Vector3 camPosition = transform.position;
+        float smoothedX = Mathf.SmoothDamp(camPosition.x, _target.x, ref _xVelocity, xSmoothTime);
+        float smoothedY = Mathf.SmoothDamp(camPosition.y, _target.y, ref _yVelocity, ySmoothTime);
```

Comment:
nit: can possibly use `Vector2.SmoothDamp` unless we need smooth time to be separate